packages/icons/src/policies.svg

```diff
@@ -0,0 +1 @@
.334 5.6096-1.627 3.0533-2.374 6.7128-2.834 9.1548-.275 1.4579-.272 4.3442-.206 7.1584.029 1.256.071 2.5027.108 3.6256.048 1.4307.089 2.6603.089 3.4523 0 .2761.224.5.5.5s.5-.2239.5-.5c0-.8027-.042-2.0652-.091-3.5173-.037-1.1228-.078-2.359-.107-3.5838-.066-2.851-.062-5.6181.189-6.9499.462-2.448 1.188-5.967 2.735-8.8697 1.54-2.8898 3.841-5.08 7.451-5.08 2.659 0 5.233 1.6418 7.163 4.243zm2.835 23.7767v-.002z" fill="#000" fill-rule="evenodd"/><path clip-rule="evenodd" d="m129.455 86.2407c-.023.2752.182.5169.457.5399 1.595.1334 3.04.2017 4.787.2565 5.936.1861 11.765.2494 17.785.2596.276.0004.5-.223.5-.4992.001-.2761-.223-.5004-.499-.5008-6.014-.0102-11.832-.0734-17.754-.2591-1.738-.0545-3.164-.1221-4.736-.2535-.275-.023-.517.1814-.54.4566z" fill="#000" fill-rule="evenodd"/><path clip-rule="evenodd" d="m135.935 72.61c-.016.2757.195.5121.471.528 4.032.2326 8.04.1788 12.057.0717.276-.0073.493-.2371.486-.5131-.007-.2761-.237-.4939-.513-.4865-4.011.1069-7.985.1596-11.973-.0705-.275-.0159-.512.1947-.528.4704z" fill="#000" fill-rule="evenodd"/><path clip-rule="evenodd" d="m141.89 63.2692c2.606.4199 5.371 3.9107 4.781 9.5326-.028.2746.171.5206.445.5495.275.0288.521-.1705.55-.4451.619-5.8965-2.268-10.0845-5.616-10.6242-1.699-.2738-3.405.4279-4.656 2.1834-1.237 1.7363-2.029 4.493-2.029 8.3893 0 .2762.224.5.5.5s.5-.2238.5-.5c0-3.7828.772-6.3056 1.843-7.809 1.058-1.4842 2.404-1.9826 3.682-1.7765z" fill="#000" fill-rule="evenodd"/><path clip-rule="evenodd" d="m142.831 79.5151c-.138.2705-.287.333-.385.333-.276 0-.5.2239-.5.5 0 .2762.224.5.5.5.617 0 1.044-.4226 1.276-.8796.235-.4621.328-1.0475.264-1.6163-.064-.5724-.292-1.1705-.749-1.6302-.465-.4682-1.129-.7539-1.982-.7539-1.24 0-2.289.7541-2.711 1.7374-.213.4979-.27 1.0645-.095 1.6186.176.5563.572 1.0566 1.186 1.4462.233.1479.542.0788.69-.1544s.079-.5422-.154-.6901c-.448-.2841-.674-.6057-.768-.9032-.095-.2998-.07-.6189.06-.9226.266-.6198.956-1.1319 1.792-1.1319.616 0 1.015.1993 1.273.4586.266.2678.42.6396.464 1.0372.046.4012-.026.7858-.161 1.0512z" fill="#000" fill-rule="evenodd"/><path clip-rule="evenodd" d="m140.002 79.8531c-.276 0-.5.2239-.5.5v3.4831l.5.0004.5-.0001v-3.4834c0-.2761-.224-.5-.5-.5zm0 4.4835c.276 0 .5-.2239.5-.5001l-.5.0001-.5-.0004c0 .2761.224.5004.5.5004z" fill="#000" fill-rule="evenodd"/><path clip-rule="evenodd" d="m142.41 79.8917c-.276 0-.5.2239-.5.5v3.5381l.5.0009.5.0001v-3.5391c0-.2761-.224-.5-.5-.5zm0 4.539c.276.0001.5-.2237.5-.4999l-.5-.0001-.5-.0009c0 .2761.224.5007.5.5009z" fill="#000" fill-rule="evenodd"/><path clip-rule="evenodd" d="m139.253 84.1492c-.088.2619.054.5451.316.6324.209.0698.417.0725.548.0742.021.0002.04.0005.056.001.365.0102.732.027 1.102.0443.022.001.044.002.066.0031.348.0162.7.0327 1.053.044.276.0089.507-.2077.516-.4837s-.208-.5069-.484-.5158c-.345-.011-.691-.0272-1.04-.0435-.022-.001-.043-.002-.064-.003-.37-.0173-.745-.0345-1.121-.045-.035-.001-.066-.0018-.092-.0024-.133-.0033-.171-.0042-.224-.0219-.262-.0873-.545.0543-.632.3163z" fill="#000" fill-rule="evenodd"/></svg>
```

Comment:
This is a big image, it should be imported as svg file directly at where it's used, it's not an icon and shouldn't be imported/used as an icon.

Reply:
it is being used at opa microfront end, i would also need at another place, instead of copy pasting this image 
i thought it would be good to export as icon